src/main.h

```diff
@@ -342,7 +346,7 @@ bool CheckTransactionWithoutProofVerification(const CTransaction& tx, CValidatio
 /** Check for standard transaction types
  * @return True if all outputs (scriptPubKeys) use only standard transaction forms
  */
-bool IsStandardTx(const CTransaction& tx, std::string& reason);
+bool IsStandardTx(const CTransaction& tx, std::string& reason, int nHeight = 0);
```

Comment:
I'm wary of this default. Shouldn't all uses now pass the height? (I know this involves updating a shedload of tests :-/)

Reply:
Yes, that's the main reason for this.  Can remove default and update tests later.

Reply:
Please file a ticket.